src/c/core/session/stream/output_reliable_stream.c

```diff
@@ -275,3 +271,14 @@ bool on_full_output_buffer(ucdrBuffer* ub, void* args)
     return false;
 }
 
+uint16_t get_available_seq_num(uxrOutputReliableStream* stream)
+{
+    int used_blocks = uxr_seq_num_sub(stream->last_written, stream->last_acknown);
+
+    used_blocks = (used_blocks == 0) ? 0 : 
+                  (used_blocks > stream->base.history) ? 
```

Comment:
possible unreachable case, since `uxr_seq_num_sub` has been fixed?

Reply:
Solved in